example_configs/lldap

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
```

Comment:
Shouldn't this file be named lldap.rc.d ? Right now it's just lldap and a bit confusing

Reply:
I think no, cause then you have to rename it back to "lldap" when you copy it over to "/usr/local/etc/rc.d/" dir

Reply:
Still, it's better to rename it to something clearer. Maybe "freebsd_lldap.rc.d", and then add in the instructions that you should rename it. Otherwise, think about it: the path to this file is "example_configs/lldap": what is it about? How can a freebsd user find the config?

Reply:
Speaking of discoverability, can you add it to the readme as well? It's not exactly a "supported integration", but it'll fit there just fine.

Reply:
@nitnelave 

If you want to add FB install instuctions into to the readme file, that's nice. With that step we should just delete them (the install instructions) from the rc.d file itself as it should not go there at the first place. 

People insterested FreeBSD lldap build can find a fully working build with the rc.d script inside included in my [temporary repo](https://github.com/n-connect/rustd-hbbx/blob/main/x86_64-freebsd_lldap-0.5.1.tar.gz). Just until a working release script/build can be included here :)

Please do make up your mind, and write me back what you'd like to close this PR. :)

Reply:
No, I was just thinking of adding a link to the (renamed) file, in the installation section (e.g. "for freebsd, you have a service file there")

Reply:
Ok. What about adding this linking into readme.md with the renamed file and have a separate text file for install instructions (freebsd_install.txt), so the rc.d script can be as it should be?

Reply:
In that case, what about a freebsd directory in example configs, with a readme and the rc script?

Reply:
Sounds great, lets do that.

One question though, what's your actual way of adding relative links into (Readme).md files? I've tried to do it in a separate repo, but never worked for me the way the [GitHub docs described](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#relative-links)

Reply:
It's worked that way for images at least, I guess for everything it would work as well.

Reply:
Ok, all done, pls. check them. The main Readme.md files internal link, I'm not sure..